core/lib/soci/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ boost_report_value(SOCI_SHARED)
 option(SOCI_STATIC "Enable build of static libraries" ON)
 boost_report_value(SOCI_STATIC)
 
-option(SOCI_TESTS "Enable build of collection of SOCI tests" ON)
+option(SOCI_TESTS "Enable build of collection of SOCI tests" OFF)

```

Comment:
First, to optimize our builds, generally we avoid rebuilding dependencies' tests, second because some dependencies' tests have side effects on some builds making them fail !